### DISCUSSION TITLE
Is there any way set Deadlock Priority?

### DISCUSSION THREAD

**Discussion by SuperArtie** [linq2db/linq2db#3436] [Q&A] [answered]
Hi all, I was wondering if there was a way to set the deadlock priority for an update statement through the Linq2DB interface. I'm just having some deadlocks pop up on a certain update and was wondering if there was something I was missing in the API documentation, that would allow me to do this like an Extension Class, etc.

Thank you!

**Comment by MaceWindu** [comment, marked as answer]
There is no deadlock management API. I don't know which database you use, but for sql server it's just an extra call like this
```cs
db.Execute("SET DEADLOCK_PRIORITY 5");
```
before update query in transaction
